CodeGeneration/Sources/generate-swiftsyntax/GenerateSwiftSyntax.swift

```diff
@@ -17,18 +17,75 @@ import Utils
 
 @main
 struct GenerateSwiftSyntax: ParsableCommand {
-  @Argument(help: "The path to the destination directory where the source files are to be generated")
+  @Argument(help: "The path to the source directory where the source files are to be generated")
   var generatedPath: String
 
   @Flag(help: "Enable verbose output")
   var verbose: Bool = false
-
+  
   func run() throws {
-    try generateTemplates(
+    try generateSwiftBasicFormat()
+    try generateIDEUtils()
+    try generateSwiftParser()
+    try generateSwiftSyntax()
+    try generateSwiftSyntaxBuilder()
+  }
+  
+  private func generateSwiftBasicFormat() throws {
+    try generateFiles(
+    templates: [
+      (basicFormatFile, "BasicFormat.swift"),
+    ],
+    inSource: "SwiftBasicFormat")
```

Comment:
What do you think about having something like:

```swift
let templates: [(builderNode: SourceFileSyntax, filename: String, module: String)] = [
  (basicFormatFile, "BasicFormat.swift", "SwiftBasicFormat"),
  (syntaxClassificationFile, "SyntaxClassification.swift", "IDEUtils")
]
```

and just generating everything from a single function?